store/tikv/gcworker/gc_worker.go

```diff
@@ -283,6 +324,22 @@ func (w *GCWorker) getOracleTime() (time.Time, error) {
 	return time.Unix(sec, nsec), nil
 }
 
+func (w *GCWorker) checkGCEnable() (bool, error) {
+	str, err := w.loadValueFromSysTable(gcEnableKey)
```

Comment:
If we load the variable failed, we will do GC. Is this what you want?

Reply:
done.